src/aten/Pow.cpp

```diff
@@ -0,0 +1,55 @@
+#include <ATen/ScalarOps.h>
+#include <ATen/XPUNativeFunctions.h>
+#include <ATen/core/Tensor.h>
+#include <ATen/native/BinaryOps.h>
+#include <ATen/native/TensorIterator.h>
+
+#include <aten/sycl/PowKernels.h>
+
+namespace at {
+
+Tensor& XPUNativeFunctions::pow_out(
+    const Tensor& base,
+    const Tensor& exp,
+    Tensor& result) {
+  auto iter = TensorIterator::binary_op(result, base, exp);
+  native::xpu::pow_tensor_tensor_kernel(iter);
+  return result;
+}
+
+Tensor& XPUNativeFunctions::pow_out(
+    const Scalar& base,
+    const Tensor& exp,
+    Tensor& result) {
+  if (base.isComplex() && base.toComplexDouble() == 1.0) {
+    result.resize_as_(exp).fill_(1);
+  } else if (!base.isComplex() && base.toDouble() == 1.0) {
+    result.resize_as_(exp).fill_(1);
+  } else {
+    XPUNativeFunctions::pow_out(
+        native::wrapped_scalar_tensor(base), exp, result);
+  }
+  return result;
+}
+
+Tensor& XPUNativeFunctions::pow_out(
+    const Tensor& self,
+    const Scalar& exponent,
+    Tensor& result) {
+  auto iter = TensorIterator::unary_op(result, self);
```

Comment:
Please align with the helpers usage,
<img width="727" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/5349567a-6566-432f-923e-cdd274bb717b">